lte/gateway/c/core/oai/tasks/ngap/ngap_state_manager.cpp

```diff
@@ -93,6 +96,56 @@ void NgapStateManager::create_state() {
   create_ngap_imsi_map();
 }
 
+status_code_e NgapStateManager::read_state_from_db() {
+#if !MME_UNIT_TEST
+  StateManager::read_state_from_db();
+#else
+  if (persist_state_enabled) {
+    magma::lte::oai::NgapState state_proto = magma::lte::oai::NgapState();
+    std::string proto_str;
+    // Reads from the AmfClientServicer DataStore Map(map_tableKey_protoStr)
+    if (NgapStateManager::getInstance().map_ngapState_tableKey_protoStr.get(
```

Comment:
define map_ngapState_tableKey_protoStr in ngap_client_servicer for UT

Reply:
Addressed review comment.